x-pack/plugins/aiops/public/components/spike_analysis_table/table_action_button.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { type FC } from 'react';
+
+import { EuiLink, EuiIcon, EuiText, EuiToolTip, type IconType } from '@elastic/eui';
+
+interface TableActionButtonProps {
+  iconType: IconType;
+  dataTestSubjPostfix: string;
+  isDisabled: boolean;
+  label: string;
+  message?: string;
```

Comment:
nit, maybe `tooltipText` or something similar would be a more descriptive name for this prop?